Snapper.NUnit/NUnitSnapper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Snapper.Core;
+using Snapper.Json;
+
+namespace Snapper.NUnit
+{
+    public class NUnitSnapper : JsonSnapper

```

Comment:
Since the user of this library would not be using this class for asserting. Should this be public?

Reply:
no, it shouldn't be, you're right.